grammar.js

```diff
@@ -138,10 +138,14 @@ module.exports = grammar(HTML, {
       '}',
     ),
 
-    _else_if_tag: _ => tag(':', 'else if'),
+    // Since 'else' and 'if' are separated by space, we cannot put them in a simple tag() like $._else_tag or $._if_start_tag
+    else_if_tag: $ => seq(
+      tag(':', 'else'),
+      field('tag', 'if'),
+    ),
     else_if_start: $ => seq(
       '{',
-      alias($._else_if_tag, $.block_tag),
+      alias($.else_if_tag, $.block_tag),
```

Comment:
not sure why the tag is exposed, that seems unnecessary. also, it might be better to have `tag` take a `rest` param that spreads any additional words after the first one. 
```
function tag(sym, text, ...other) {
  return seq(
    sym,
    field('tag', token.immediate(text)),
    ...other.map(rule => field('tag', rule)),
  );
}
``` should work, and this way you can just put `'if'` after `'else'` in the tag call.